### DISCUSSION TITLE
Welcome to name-suggestion-index Discussions!

### DISCUSSION THREAD

**Discussion by bhousel** [osmlab/name-suggestion-index#4786] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by bhousel** [comment]
Just trying out the new GitHub Discussions features.  This might be a good way for us to discuss things where we don't necessarily need an issue?

**Reply by michaelblyons** [reply]
Maybe the [question]-tagged issues belong here? I'm not asserting one way or another. It's just a possibility.

**Reply by bhousel** [reply]
Yeah I think so!  I will probably move a few issues over here and see what happens.
I'm still figuring out how/if we want to use it.  This new feature just landed in GitHub I think yesterday.
